tests/ActivityloggerTest.php

```diff
@@ -100,25 +100,6 @@ public function it_can_log_an_activity_with_a_causer()
     }
 
     /** @test */
-    public function it_can_log_an_activity_with_a_causer_when_there_is_no_web_guard()
-    {
-        config(['auth.guards.web' => null]);
```

Comment:
Well  web guard will always be the default in laravel , i dont understand why would it be null ...
and when it would be null unless you deleted it...